integration/tests/cook/test_basic.py

```diff
@@ -892,45 +921,46 @@ def jobs_query():
 
             util.wait_until(jobs_query, util.all_instances_done)
             # retry all jobs in the group
-            util.retry_jobs(self.cook_url, retries=2, groups=[group_uuid])
+            util.retry_jobs(self.cook_url, retries=2, groups=[group_uuid], **extra_retry_args)
```

Comment:
Can we just inline the args here instead of creating the separate `extra_retry_args` variable?

Reply:
Yes. I think this is an artifact of refactoring an earlier version of the test. This should definitely be inlined.